src/studio/src/designer/backend/Controllers/ConvertV1TextsToV2Controller.cs

```diff
@@ -0,0 +1,49 @@
+using System;
```

Comment:
No need to have this as a separate controller as the action is related to texts resources. Suggest to move this to:
`designer/api/v2/{org}/{repo}/texts/convert` the versions are implicit as we only have v1 and v2.